OpenIPSL/Interfaces.mo

```diff
@@ -0,0 +1,89 @@
+within OpenIPSL;
+package Interfaces
+  "Package for interface models used in this library. "
+  extends Modelica.Icons.InterfacesPackage;
+  connector PwPin "Connector for electrical blocks treating voltage and current as complex variables"
```

Comment:
@dietmarw shouldn't you save this package as separate files? 
(if not, let me know why)